src/language-js/parse-postprocess.js

```diff
@@ -3,21 +3,12 @@
 const {
   getLast,
   getNextNonSpaceNonCommentCharacter,
-  getShebang,
```

Comment:
Didn't you forget to delete this function in `common/util`? Or is it still used?

Reply:
Needed, https://github.com/prettier/prettier/blob/fc396277fc64ca6d056960fb43e9b1ad5296f597/src/language-js/pragma.js#L8 and   https://github.com/prettier/prettier/blob/fc396277fc64ca6d056960fb43e9b1ad5296f597/src/language-js/parser-babel.js#L61